measures/ResStockArguments/measure.rb

```diff
@@ -379,6 +379,9 @@ def run(model, runner, user_arguments)
 
     args_to_delete = args.keys - arg_names # these are the extra ones added in the arguments section
 
+    # Set operational calculation
+    args['occupancy_calculation_type'] = HPXML::OccupancyCalculationTypeOperational
```

Comment:
@joseph-robertson I don't know that there's a compelling use case for exposing this in options_lookup.tsv, but you're welcome to do so if you wish.

Reply:
I'll leave that up to @ejhw.

Reply:
If there is a use case for exposing this input, we can do so easily in a subsequent PR.

Reply:
It's been a while since we've built an ECE run. It was on a much older version of ResStock. I could see this being potentially useful to have a switch for asset calculations. I say add it if it doesn't cause too much touble.